rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -70,6 +70,45 @@ THE SOFTWARE.
 
 #define MAX_ASPECT_RATIO 3.0f
 
+void get_rocal_tensor_layout(RocalTensorLayout &tensor_layout, RocalTensorlayout &op_tensor_layout)
+{
+    switch(tensor_layout)
```

Comment:
What is the purpose of this api. We are just copying one tensor_layout to other?